TODO.md

```diff
@@ -0,0 +1,32 @@
+# CNCF Hugo Starter Checklist
+
+## CNCF Website Guidelines
+
+As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md):
+
+- [ ] 1. Website should be hosted in an open source repo
+  - [ ] Hosted in the same organization as the main project
+- [ ] 2. Reference the origin company correctly
+  *Note*: It is OK to say that, e.g., “Prometheus was originally created by Soundcloud” or “Kubernetes builds upon 15 years of experience of running production workloads at Google,” but the origin company should not otherwise be referred to on the project homepage.
+- [ ] 3. No links or forms for capturing enterprise support leads present.
```

Comment:
It might be worth restructuring this document into two sections/paths:

- If you already have a website coming into the CNCF, here's how to bring it into compliance
- If you don't have a website and need one, here's the baseline

This is a good todo, but only for some users?